lib/roadie/rails/asset_pipeline_provider.rb

```diff
@@ -18,7 +18,13 @@ def find_stylesheet(name)
 
       private
       def normalize_asset_name(href)
-        remove_asset_prefix href.split('?').first
+        res = remove_asset_prefix href.split('?').first
```

Comment:
Oh, sorry. Looking at it now makes it obvious that you didn't add anything on that line. I must've been very tired or something. :flushed: